pkg/pool-manager/pod_pool.go

```diff
@@ -28,13 +28,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const tempPodName = "tempPodName"
```

Comment:
it is possible. by the time the controller is called - they pod must have a name
so if 2 pods try to get the same mac and end up not having names in the KMP webhook context - we won't be able to prevent collision - I agree
I think I can solve it by only allowing 1 temp pod name to be set at a time.

Reply:
I see. Thanks for explaining